test/debug_info/lit.local.cfg

```diff
@@ -0,0 +1,11 @@
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+#debug info test configuration
+
+config.suffixes = ['.f', '.FOR', '.for', '.f77', '.f90', '.f95', '.F', '.fpp',
+ '.FPP', '.ll', '.o']
```

Comment:
is ll and o required?

Reply:
Thanks @kiranchandramohan for reviewing this.
> is ll and o required?

I added them to be on safer side, but I guess we might only be verifying the **IR** and in-case of **dwarf** we can pipe the intermediate object to **llvm-dwarfdump** and verify that. I'll remove them in next iteration, later if things demands we can add them in.

> Is a makefile required?

No (I may be wrong here), Since this is up and running.

> Are the temp files cleaned up?

I did notice 3 directories getting created in build area inside **test/debug_info** 
(**lit, Output, src**) of which **Output** is only relevant(Considering LLVM style **lit**), might need to clean up these 2 directories.
In **Output** directory only script files per test case(i.e pass.f90.script) remains **no temp files**.  This behavior is analogous to LLVM lit semantics.

Reply:
> I did notice 3 directories getting created in build area inside test/debug_info
(lit, Output, src)

Just wanted to add a correction here, that these were part of an unclean build. Only **Output** directory is getting created, with script files as mentioned above.